src/Service/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -0,0 +1,708 @@
+
+            if (includePrimaryKeyPathComponent)
+            {
+                Tuple<string, List<OpenApiParameter>> pkComponents = CreatePrimaryKeyPathComponentAndParameters(entityName);
+                string pkPathComponents = pkComponents.Item1;
+                string fullPathComponent = entityBasePathComponent + pkPathComponents;
+
+                OpenApiOperation getOperation = new()
+                {
+                    Description = GETONE_DESCRIPTION,
+                    Tags = tags,
+                    Responses = new(_defaultOpenApiResponses),
+                };
+                getOperation.Responses.Add(HttpStatusCode.OK.ToString("D"), CreateOpenApiResponse(description: nameof(HttpStatusCode.OK), responseObjectSchemaName: entityName));
+
+                OpenApiOperation putOperation = new()
+                {
+                    Description = PUT_DESCRIPTION,
+                    Tags = tags,
+                    Responses = new(_defaultOpenApiResponses),
+                    RequestBody = CreateOpenApiRequestBodyPayload(schemaReferenceId, requestBodyRequired)
+                };
+                putOperation.Responses.Add(HttpStatusCode.OK.ToString("D"), CreateOpenApiResponse(description: nameof(HttpStatusCode.OK), responseObjectSchemaName: entityName));
+                putOperation.Responses.Add(HttpStatusCode.Created.ToString("D"), CreateOpenApiResponse(description: nameof(HttpStatusCode.Created), responseObjectSchemaName: entityName));
+
+                OpenApiOperation patchOperation = new()
+                {
+                    Description = PATCH_DESCRIPTION,
+                    Tags = tags,
+                    Responses = new(_defaultOpenApiResponses),
+                    RequestBody = CreateOpenApiRequestBodyPayload(schemaReferenceId, requestBodyRequired)
+                };
+                patchOperation.Responses.Add(HttpStatusCode.OK.ToString("D"), CreateOpenApiResponse(description: nameof(HttpStatusCode.OK), responseObjectSchemaName: entityName));
+                patchOperation.Responses.Add(HttpStatusCode.Created.ToString("D"), CreateOpenApiResponse(description: nameof(HttpStatusCode.Created), responseObjectSchemaName: entityName));
+
+                OpenApiOperation deleteOperation = new()
+                {
+                    Description = DELETE_DESCRIPTION,
+                    Tags = tags,
+                    Responses = new(_defaultOpenApiResponses),
+                    RequestBody = CreateOpenApiRequestBodyPayload(schemaReferenceId, requestBodyRequired)
+                };
+                deleteOperation.Responses.Add(HttpStatusCode.NoContent.ToString("D"), CreateOpenApiResponse(description: nameof(HttpStatusCode.NoContent)));
+
+                OpenApiPathItem openApiPathItem = new()
+                {
+                    Operations = new Dictionary<OperationType, OpenApiOperation>()
+                    {
+                        // Creation GET, POST, PUT, PATCH, DELETE operations
```

Comment:
Missing POST?

Reply:
Looks like the comment should be fixed. POST is in the else part

Reply:
fixed